src/store/migrations.js

```diff
@@ -0,0 +1,21 @@
+export const migrations = {
+  1: ({_persist}) => {
+    return {_persist}; // reset all state, except version
+  },
+  2: (state) => {
+    return {
+      ...state,
+      catalogos: undefined,
+      countdown: undefined,
+      calendar: undefined,
+    }; // reset all state, except version
+  },
+  3: (state) => {
+    return {
+      ...state,
+      catalogos: undefined,
+      countdown: undefined,
+      calendar: undefined,
+    }; // reset all state, except version
```

Comment:
why are `2` and `3` the same?

Reply:
Cuz we made the tests with v2, and i added v3 just in case ;3